<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>127</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_right
{
  border:3px solid white;
  min-width:40px;
  max-width:40px;
  border-radius:23px;
}
QWidget#widget_bottom
{
border:3px solid white;
border-radius:23px;
min-width:40px;
max-width:40px;
min-height:40px;
max-height:40px; 
background-color: transparent;
}
QWidget#widget_top
{
border:3px solid white;
border-radius:23px;
min-width:40px;
max-width:40px;
min-height:40px;
max-height:40px; 
background-color: transparent;
}

QPushButton
{
min-height:40px;
min-width:40px;
max-height:40px;
max-width:40px;
border-radius:20px;
border:0px solid white;
background-color: rgb(0, 85, 255);
font:italic bold 20px arial;
color:white;
}
QPushButton:pressed
{
background-color: rgb(85, 255, 255);
}
QLabel#label_top
{
background-color:transparent;
color:white;
}
QLabel#label_bottom
{
background-color:transparent;
color:white;
}
QLabel#label_center_1
{
background-image: url(:/arrow-enable1.png);
min-height:20px;
min-width:20px;
max-height:20px;
max-width:20px;
}
QLabel#label_center_2
{
background-image: url(:/arrow-enable1.png);
min-height:20px;
min-width:20px;
max-height:20px;
max-width:20px;
}
QLabel#label_flag
{
background-color:white;
min-height:3px;
max-height:3px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>60</number>
     </property>
     <property name="bottomMargin">
      <number>60</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_left_1">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_left_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>97</number>
     </property>
     <property name="bottomMargin">
      <number>96</number>
     </property>
     <item row="1" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_center_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_center_1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_right" native="true">
     <widget class="QWidget" name="widget_bottom" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>46</width>
        <height>46</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_bottom">
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_top" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>46</width>
        <height>46</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="label_flag">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>194</y>
        <width>46</width>
        <height>3</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_top">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>46</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string>300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
